### PR TITLE
Merge 508 and e2e tests for 1990 and 1995 edu forms

### PR DESCRIPTION
**Pull request by raquelromano-gov**
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2160
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/2203
Connects to https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1840

### CODE REVIEW

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js [hunk 1]
```diff
@@ -89,14 +104,15 @@ module.exports = E2eHelpers.createE2eTest(
...
     client.expect.element('.js-test-location').attribute('data-location')
       .to.not.contain('/review-and-submit').before(Timeouts.submission);
 
     // Confirmation page.
     client
       .expect.element('.edu-benefits-submit-success').to.be.visible;
-
-    client.end();
+    client.axeCheck('.main')
+          .end();
```
Comment (jbalboni) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: Indentation seems weird here?

**Review by jbalboni — APPROVED** [review]
LGTM

File: test/edu-benefits/1990/01-all-fields.e2e.spec.js [hunk 2]
```diff
@@ -1,128 +1,156 @@
...
       .expect.element('input[name="seniorRotcCommissioned-0"]').to.be.present;
     EduHelpers.completeRotcHistory(client, EduHelpers.testValues, false);
-    client.click('.form-progress-buttons .usa-button-primary');
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
     E2eHelpers.expectNavigateAwayFrom(client, '/military-history/rotc-history');
 
-    // Benefits history
+    // Benefits history page
```
Comment (annekainicUSDS) [inline test/edu-benefits/1990/01-all-fields.e2e.spec.js]: Missed this page perhaps with the `.axeCheck`?

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js [hunk 3]
```diff
@@ -37,28 +40,36 @@ module.exports = E2eHelpers.createE2eTest(
...
     client
       .waitForElementVisible('label[for="root_oldSchool_name"]', Timeouts.slow);
     Edu1995Helpers.completeOldSchool(client, testData.oldSchool.data, false);
-    client.click('.form-progress-buttons .usa-button-primary');
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
     E2eHelpers.expectNavigateAwayFrom(client, '/school-selection/old-school');
 
     // Contact information page.
```
Comment (annekainicUSDS) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: Do we also want to do the `.axeCheck` for this page?

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js [hunk 4]
```diff
@@ -23,7 +24,9 @@ module.exports = E2eHelpers.createE2eTest(
     client
       .waitForElementVisible('input[name="root_veteranFullName_first"]', Timeouts.slow);
     EduHelpers.completeVeteranInformation(client, testData.veteranInformation.data, false);
-    client.click('.form-progress-buttons .usa-button-primary');
+    client
+      .axeCheck('.main')
+      .click('.form-progress-buttons .usa-button-primary');
     E2eHelpers.expectNavigateAwayFrom(client, '/veteran/information');
 
     // Benefits eligibility page.
```
Comment (annekainicUSDS) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: `.axeCheck` missing here too perhaps?

**Review by annekainicUSDS — COMMENTED** [review]
Looks good! Perhaps a couple places where we still need to do the `.axeCheck`?

File: test/edu-benefits/1990/01-all-fields.e2e.spec.js (hunk 2 above)
Comment (raquelromano-gov) [inline test/edu-benefits/1990/01-all-fields.e2e.spec.js]: Oops, thanks!

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js (hunk 4 above)
Comment (raquelromano-gov) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: Fixed

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js (hunk 3 above)
Comment (raquelromano-gov) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: Fixed

File: test/edu-benefits/1995/01-all-fields.e2e.spec.js (hunk 1 above)
Comment (raquelromano-gov) [inline test/edu-benefits/1995/01-all-fields.e2e.spec.js]: Fixed

**Comment by raquelromano-gov** [comment]
Oops, bad merge...fixing

**Review by cvalarida — APPROVED** [review]

**Pull request merged**